openhands/runtime/client/runtime.py

```diff
@@ -289,7 +289,7 @@ def _init_container(
                 name=self.container_name,
                 detach=True,
                 environment=environment,
-                volumes=volumes,
+                volumes=None,
```

Comment:
We need to keep this, but should only set `volumes` if the right env vars are set (e.g. `workspace_mount_path`)